main/ZmqttDiscovery.ino

```diff
@@ -125,7 +128,32 @@ void createDiscovery(char* sensor_type,
     device.set("sw_version", OMG_VERSION);
     JsonArray& identifiers = device.createNestedArray("identifiers");
     identifiers.add(getMacAddress());
-    sensor.set("device", device); //device sensor is connected to
+    sensor.set("device", device); //device representing the board
+  } else {
+    char deviceid[13];
+    memcpy(deviceid, &unique_id[0], 12);
+    deviceid[12] = '\0';
+    StaticJsonBuffer<JSON_MSG_BUFFER> jsonDeviceBuffer;
+    JsonObject& device = jsonDeviceBuffer.createObject();
+    if (device_mac != "") {
+      JsonArray& connections = device.createNestedArray("connections");
+      JsonArray& connection_mac = connections.createNestedArray();
+      connection_mac.add("mac");
+      connection_mac.add(device_mac);
+    }
+    JsonArray& identifiers = device.createNestedArray("identifiers");
+    identifiers.add(deviceid);
+    if (device_manufacturer) {
```

Comment:
You can check it with this:
`if(device_manufacturer[0])`

Reply:
fixed